.travis.yml

```diff
@@ -1,38 +1,52 @@
 language: c
-script: make && make test # no ./configure
+script: meson build && cd ./build && ninja
 
 # Uncomment me if this gets annoying
 #
-# notifications: 
-#   email: false 
+# notifications:
+#   email: false
+
+before_install:
+  - wget https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip
+  - unzip ninja-linux.zip
+  - sudo mv ninja /usr/bin/
+  - wget https://github.com/libuv/libuv/archive/v1.19.2.tar.gz
```

Comment:
can we submodule `libuv` in properly as a fallback? I get the unbundling thing, but if we're downloading and compiling it anyway on LTE this should be represented in the git directory structure

Reply:
There's a libuv1 package available for Xenial (https://packages.ubuntu.com/xenial/libuv1), but nothing new enough for Trusty (https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=libuv&searchon=names).